torch/onnx/_internal/onnxruntime.py

```diff
@@ -887,9 +887,9 @@ def maybe_map_to_meta_val(value):
                 )
             else:
                 try:
-                    prim_outputs = FakeTensorProp(graph_module).propagate(
-                        *args, **kwargs
-                    )
+                    prim_outputs = FakeTensorProp(
+                        graph_module, check_consistency=False
+                    ).propagate(*args, **kwargs)
```

Comment:
Any reason why we wouldn't want to check consistency here?

Reply:
Well, it fails your unit tests, and I couldn't figure out what ONNX was doing lol. The relevant code is `python test/onnx/dynamo/test_dynamo_with_onnxruntime_backend.py -k test_elementwise_function_multiple_output_1

Reply:
We will track this on https://github.com/pytorch/pytorch/issues/124173